src/Config/ShieldOAuthConfig.php

```diff
@@ -42,7 +42,8 @@ class ShieldOAuthConfig extends BaseConfig
             'client_id'     => 'Get it from Google',
             'client_secret' => 'Get it from Google',
 
-            'allow_login' => true,
+            'allow_login'    => true,
+            'allow_register' => false,
```

Comment:
Registration is currently open. If this value is set to `false`, it will cause programs to fail. The default value of `true `seems appropriate.

Reply:
this one has been fixed